libs/logging/src/logger.ts

```diff
@@ -43,6 +44,14 @@ export class Logger {
       disposition,
       ...additionalData,
     };
+    // If the caller is passing in a debug instance, and we are not in production log to the debugger rather then through the normal logging pipeline.
+    // This is to make logs more manageable in development, so a developer can toggle what logs to view with the normal debug namespaces.
+    /* istanbul ignore next - figure out how to test this */
+    if (outerDebug && process.env.NODE_ENV !== 'production') {
+      outerDebug(logLine);
```

Comment:
I'm unsure how to feel about this. Part of me feels it should just be that the `logger` used is one that uses `debug` or doesn't depending on the environment, rather than mixing the two. I don't feel strongly enough to push back on it in this PR, but I do think we should watch how this works for us and change if it becomes unwieldy.